vital/algo/dynamic_configuration.cxx

```diff
@@ -2,11 +2,13 @@
 // OSI-approved BSD 3-Clause License. See top-level LICENSE file or
 // https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
 
-#include <vital/algo/dynamic_configuration.h>
 #include <vital/algo/algorithm.txx>
+#include <vital/algo/dynamic_configuration.h>
```

Comment:
Directly corresponding headers (i.e. `foo.h`, `foo_p.h` in `foo.cxx`) should be first, and separated by a blank line from other headers. I see this elsewhere:
```suggestion
#include "dynamic_configuration.h"

#include <vital/algo/algorithm.txx>
```
Not sure what our convention is for `""` includes vs. `<>` includes for such headers, but in any case, please add the blank line.

Reply:
@mwoehlke-kitware Did you ever determine what the "" vs <> convention is?

Reply:
I don't see a Kwiver convention mentioned [here](https://docs.google.com/document/d/1I5gP61K2VuV1Ki10uwpK0zd6o--4wR-uDtidl33ML0g/edit#) or [here](https://github.com/Kitware/kwiver/blob/master/doc/code-style.rst).  Perhaps there is a broader Kitware standard we could find and apply, but so far I'm not finding one.